File: UI/pinguino/src/app/models/produto.ts
export class Produto {

    constructor() {
        this.id = 0;
        this.nome = '';
        this.sku = '';
        this.codigobarras = '';
        this.fornecedor_id = 0;
        this.fornecedor_nome = '';
        this.descricao = ''
        this.precovenda = 0.0;
        this.ativo = true;
    }

    id: number;
    nome: string;
    sku: string;
    codigobarras: string;
    fornecedor_id: number;
    fornecedor_nome: string;
    descricao: string;
    precovenda : number
    ativo: boolean;
}

File: API/PinguinoApp/PinguinoApp.API/Models/Produto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PinguinoApp.API.Models
{
    public class Produto
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string Sku { get; set; }
        public string Codigobarras { get; set; }
        public Fornecedor Fornecedor { get; set; }
        public string Descricao { get; set; }
        public decimal PrecoVenda { get; set; }
        public bool Ativo { get; set; }


    }
}
